src/main/java/org/myrobotlab/service/Agent.java

```diff
@@ -1220,8 +1254,8 @@ public static void main(String[] args) {
         agent.shutdown();
       }
 
-      if (options.manifest) {
-        Map<String, String> manifest = Runtime.getManifest();
+      if ("".equals(options.version)) {

```

Comment:
seems like a StringUtils.isEmpty(options.version) would be better here,  I wonder could options.version be null?

Reply:
yes, it can be null, but not when the flag was supplied by the user and  the arity=0..1 is in its definition @Option(names = { "-v", "--version" }, arity = "0..1"